src/app/result-details/result-details.component.html

```diff
@@ -26,8 +26,7 @@ <h1>{{clinicalTrial.title}}</h1>
       </dl>
       <div class="status-icons">
         <div class="overall-status">
-          <div class="icon">{{clinicalTrial.overallStatus}}</div>
-          <div class="label">{{clinicalTrial.overallStatus}}</div>
+          <div class="icon" [ngStyle]="{'background': trialStatusColor(clinicalTrial.overallStatus)}">{{statusDisplay[clinicalTrial.overallStatus]}}</div>
```

Comment:
Rather than set the background color based on JavaScript code, it would probably make more sense to set a CSS class based on the overall status, after verifying that the returned status is valid. Then the colors can be declared in the CSS file rather than in JavaScript.

Reply:
Moved colors to css